Adds benchmarking tests (#12)
aguirre-lab/aguirre-lab: README.md

ERISOne is the MGB research computing cluster. We do not usually use it, but have documented how to set it up for a data science workflow: [`erisone documentation`](/erisone.md)

### Workstation benchmarking tests

> Workstation's RAM, CPU and GPU can be tested with stress tests. These tests put the workstation under load and log the system status.
> 
> GPU, CPU and RAM usage as well as power consumption and heat load are logged into files.
> 
> This tests are recommended when there is a belief that the machine might be crashing due to hardware failure.
> 
> In order to run the tests and analyze the system status, follow these steps:

Reviewer: ```suggestion
Test and log your workstation's RAM, CPU, and GPU usage, power consumption, and heat load.

This can be helpful if a machine is crashing and you suspect a hardware failure.

To run the benchmark tests:
```

1. Install all dependencies and tools:
    ```bash

>     $ cd workstation_benchmarking_tests
>     $ ./install_benchmarks.sh

Reviewer: ```suggestion
    $ git clone git@github.com:aguirre-lab/aguirre-lab.git
    $ cd aguirre-lab/workstation_benchmarking_tests
    $ ./install_benchmarks.sh
```

    ```

>     It will ask for several questions when finding the thermal sensors. Say `yes` to all of them.

Reviewer: ```suggestion
    You will be asked several questions as the script finds thermal sensors. Say `yes` to all of them.
```

>     Finally, it will ask if there is GPU. Say `yes` or `no` depending on your workstation.

Reviewer: ```suggestion
    Finally, you will be asked if you have a GPU. Say `yes` or `no` depending on your workstation.
```

    
2. Run the desired tests:
    ```bash
    $ cd workstation_benchmarking_tests
    $ ./run_benchmarks.sh
    ```
   It will ask for the tests you want to run as well as how many time you want them to last. Possible options for tests are:

>    `ram`, `cpu`, `gpu` and `all`.

Reviewer: This is really nice!

> 3. Find results of thermal load, power consumption and GPU, CPU and RAM usage in `./workstation_benchmarking_tests/result_logs`.

Reviewer: ```suggestion
3. Results of GPU, CPU and RAM usage, thermal load, and power consumption will be logged in `./workstation_benchmarking_tests/result_logs`.
```
